src/scirpy/ir_dist/__init__.py

```diff
@@ -83,22 +83,20 @@ def _get_distance_calculator(metric: MetricType, cutoff: Union[int, None], *, n_
         metric = "identity"
         cutoff = 0
 
-    # Let's rely on the default set by the class if cutoff is None
-    if cutoff is not None:
-        kwargs["cutoff"] = cutoff
-
```

Comment:
This was actually intentional, that if `None` is passed to `_get_distance_calculator`, **no cutoff parameter** is passed to the respective class. Like that the default value defined within the class is chosen.

Reply:
This was done to make the default values clearer in the documentation, while maintaining API compatibilty of the user-facing functions. (see also https://github.com/scverse/scirpy/issues/471)